draft-ietf-dprive-dnsoquic.md

```diff
@@ -619,6 +620,15 @@ to renew session resumption tickets.
 Servers SHOULD implement the anti-replay mechanisms specified in section 8 of
 {{?RFC8446}}.
 
+### Controlling Connection Migration For Privacy
+
+DoQ implementation might consider using the connection migration features defined
+in section 9 of {{!RFC9000}}. These features enable connections to continue operating
```

Comment:
```suggestion
in Section 9 of {{!RFC9000}}. These features enable connections to continue operating
```

Of course "{{Section 9 of RFC9000}}" is better, but you can't use the ! with that (yet, or to my knowledge).